splice_machine/src/test/java/com/splicemachine/derby/utils/ShowCreateTableIT.java

```diff
@@ -618,4 +618,2118 @@ private static void  checkEqualIgnoreConstraintOrder(String expected, String act
         String expectedResult = Stream.of(expected.split("CONSTRAINT ")).map(str -> str.substring(0, str.length() - 2)).sorted().collect(Collectors.joining(", "));
         Assert.assertEquals(expectedResult, result);
     }
+
+    @Test
+    public void testTable2100Columns() throws Exception {
+        String ddl = "CREATE TABLE \"" + SCHEMA + "\".\"TEST_2100\" (\n" +
+                "\"COL_0\" VARCHAR(30)\n" +
+                ",\"COL_1\" CHAR(10)\n" +
+                ",\"COL_2\" BIGINT\n" +
+                ",\"COL_3\" DATE\n" +
```

Comment:
can you write a for-loop generating this? i guess you already have, or did you create 2100 columns by hand?

Reply:
I do have a program generating it, but it randomize the column types...Also, we have to generate the DDL as the way how `show create table` formats it, so that it can be used for both creating and comparing. I'll tune my current code to do it.